chapters/1.md

```diff
@@ -3,37 +3,63 @@
+
+Leo and George's transaction probably wouldn't be done via Fedwire because it has large cost. It might be done via SWIFT or ACH or check or some other deferred settlement system, but I imagine if they *could* transfer instantly and immediately they would prefer to do so without involving a third party.
+
+So the real problem is that given the distance at which they are transacting, there are layers upon layers of trust that Leo must have in the system that he is using just to give his friend George enough money to take a nice vacation. Leo must trust that the bank won't abscond with his money that he has given them to hold, George's bank must trust that Leo's bank will give them the money if they allow Leo to have it and won't reverse the transaction, or the banks have to trust the central bank to transfer the money between their accounts.
+
+In a modern functioning country and economy, some of these things are pretty ridiculous to imagine, but in the global picture, there aren't always the kinds of protections (like deposit insurance), or systems in place free from corruption. There are banks, and businesses, and people who can't rely on their government and shady business practices, and banks that go out of business. There's no guarantee that the systems that remain in place will continue to do so and operate in a manner that is fair for all participants. There is an incredible amount of complexity in our financial system with lots of room for problems to arise. These layers of trust and steps of verification add cost, and delay to Leo and George's transaction.
+
+What does this have to do with cryptocurrency? What if Leo could send money to George directly, in a trustless way, in a way that could not be reversed, with minimal delay, that doesn't require the involvement of a bank, or even banks, or government? How about Chase and USAA; how can they transact and settle their accounts without having to trust each other and without having to pay a third party like the SWIFT network or the central bank for the privilege of doing business with each other while retaining as much liquidity as possible and reducing costs, for their profit and to the benefit of their customers and shareholders?
+
+To do that, banks and individuals can use something called blockchain. Blockchain is a technology that is a distributed database that continuously adds to a record of all the transactions that have occurred within a network. This database is often referred to as a ledger, because the data contains a large list of transactions that have taken place.
```

Comment:
I think we should emphasize here that "blockchain is based on cryptographic rules which can verify each transaction that is linked with an hash." I know I have to write more than this, but I would like to mark the concept. 

At the moment, Home Banking works with a database. Is that database a blockchain? If we allow the definition that is currently on the resource, yes. But technically it isn't.

Reply:
I can make an attempt to clarify blockchain here, but I don't know where in the book you were going to flesh it out.

Reply:
Ah, you can mention the cryptography.